server/schemas-to-ts/interface-builders/interfaceBuilder.ts

```diff
@@ -254,19 +250,20 @@ export abstract class InterfaceBuilder {
 
     let interfaceText = `export interface ${interfaceName} {\n`;
     if (schemaInfo.source === SchemaSource.Api || schemaInfo.source === SchemaSource.Extension) {
-      interfaceText += `  id: number;\n`;
+      interfaceText += `  documentId: string;\n`;
     }
+    if (interfaceName === 'Article') this.commonHelpers.logger.information(schemaInfo)
 
     let indentation = '  ';
     if ((schemaInfo.source === SchemaSource.Api || schemaInfo.source === SchemaSource.Extension) && schemaType === SchemaType.Standard) {
-      interfaceText += `  attributes: {\n`;
-      indentation += '  ';
+      // interfaceText += `  attributes: {\n`;
+      // indentation += '  ';
```

Comment:
okay, maybe clean this up and remove these lines instead of commenting them out :)